docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -627,9 +596,78 @@ simplifying assumptions that we have included:
 can use an interactive simulation in a Jupyter Notebook to check that
 simulants are moving at the expected rates.
 
+.. _census_prl_international_immigration:
 
-2.3.5 Component 9: Address
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.3.5 Component 9: International Immigration (In-Migration)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New simulants are added by migration into the US from other countries.
+We simulate two kinds of immigration: household moves and non-reference-person moves.
+
+#. A **household move** is when an entire household (which may be a single person) enters from outside the country as a unit,
+   preserving relationships within the unit.
+#. A **non-reference-person move** is when a person enters from outside the country and joins an existing household.
+   Non-reference-person moves are independent, single-person events that do not preserve relationship structure.
+
+For the purposes of immigration, a group quarters person should be treated identically to a one-person household,
+with the group quarters person as the reference person.
+This means that group quarters people should only enter the US in a household move.
+
+The number of simulants who move to the US each year is informed by the ACS' "residence one year ago" question.
+Specifically, a value of 2 for variable :code:`MIG` indicates that a respondent lived outside the US one year ago,
+while any other value indicates that they lived within the US.
+We refer to respondents who were not living in the United States one year ago as "recent immigrants."
+Our assumption is that the number (and characteristics) of recent immigrants
+in the 2019 ACS will be replicated in each future year.
+
+We also assume that the proportions of recent immigrants by move type (household or non-reference-person) will remain constant.
+Though in reality not all moves into the US follow one of these two patterns, we assume that any new immigrant in a household
+where the reference person is also a new immigrant was part of a household move, while any new immigrant in a household where the
+reference person is not a new immigrant moved to the US in a non-reference-person move.
+
+Specifically, the yearly rate at which simulants are added to the population by each move type is given by
+the (weighted) proportion of ACS PUMS persons that are recent immigrants consistent with that move type.
+At each time step:
+
+#. PUMS households with reference people who are recent immigrants, after removing household members who are not recent immigrants,
+   are sampled with replacement until the desired number of simulants added in household moves is reached.
```

Comment:
if we aim for the desired number of simulants added in household moves, do we have concerns about the number of recent immigrant households being quite far off from the data?

Reply:
I would not expect this to happen. It would require the household size of the sampled households to be systematically different from the households in ACS with a recent immigrant reference person. Since that is precisely the group we will be sampling from, the household size should be the same, modulo some sampling error. Or am I missing something?